### DISCUSSION TITLE
Adoption of New Academic Terminology and how it could affect CoDoc

### DISCUSSION THREAD

**Discussion by harin0826** [AY2223S2-CS2103T-F12-2/tp#26] [General]
Hello guys, I'm sure you are all aware about the recent changes in academic terminologies within NUS (released 13 Feb 2023):
(a)	“Module” will be renamed “Course”.
(b)	“Modular Credit (MC)” will be renamed “Unit”.
(c)	“Cumulative Average Point (CAP)” will be renamed “Grade Point Average (GPA)”.

This could mean that our current "course of study" may no longer mean "Computer Science" or "Information Security", and our current "module" may not be relevant.

What are your thoughts on this? Should we be changing our product to fit the upcoming changes?

My stand is that since this change will only apply from 1 August 2023, there is no need to apply these changes, but we should keep these changes in mind to improve sustainability of our product.

**Comment by linustws** [comment]
Ya I think we shouldn't change for now, if we change it will be confusing for our audience to understand when we present our product.
